repository/src/main/kotlin/com/melih/repository/sources/NetworkSource.kt

```diff
@@ -1,18 +1,11 @@
 package com.melih.repository.sources
 
-import android.net.NetworkInfo
+import android.net.ConnectivityManager
 import com.melih.repository.DEFAULT_IMAGE_SIZE
 import com.melih.repository.Repository
 import com.melih.repository.entities.LaunchEntity
 import com.melih.repository.interactors.DEFAULT_LAUNCHES_AMOUNT
-import com.melih.repository.interactors.base.EmptyResultError
-import com.melih.repository.interactors.base.Failure
-import com.melih.repository.interactors.base.NetworkError
-import com.melih.repository.interactors.base.Reason
-import com.melih.repository.interactors.base.ResponseError
-import com.melih.repository.interactors.base.Result
-import com.melih.repository.interactors.base.Success
-import com.melih.repository.interactors.base.TimeoutError
+import com.melih.repository.interactors.base.*
```

Comment:
Wildcard imports are discouraged, can you import whats used ? This will fail CQ too.

Reply:
Sure. Done